src/main/controllers/ClaimantEmploymentDatesController.ts

```diff
@@ -51,7 +51,7 @@ export default class ClaimantEmploymentDatesController {
 
   public post = async (req: AppRequest, res: Response): Promise<void> => {
     const nextPagae =
```

Comment:
Minor nitpick - will need changing on line 63 if you do fix it
```suggestion
    const nextPage =
```